File: sample/src/main/java/contacts/sample/cheatsheet/other/java/DebugContactsProviderTablesActivity.java
package contacts.sample.cheatsheet.other.java;

import android.app.Activity;

import contacts.debug.*;

public class DebugContactsProviderTablesActivity extends Activity {

    void debugContactsProviderTables() {
        LogGroupsTableKt.logGroupsTable(this);
        LogAggregationExceptionsTableKt.logAggregationExceptionsTable(this);
        LogProfileKt.logProfile(this);
        LogContactsTableKt.logContactsTable(this);
        LogRawContactsTableKt.logRawContactsTable(this);
        LogDataTableKt.logDataTable(this);
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/other/kotlin/DebugContactsProviderTablesActivity.kt
package contacts.sample.cheatsheet.other.kotlin

import android.app.Activity
import contacts.debug.*

class DebugContactsProviderTablesActivity : Activity() {

    fun debugContactsProviderTables() {
        logGroupsTable()
        logAggregationExceptionsTable()
        logProfile()
        logContactsTable()
        logRawContactsTable()
        logDataTable()
    }
}